src/LibraryViewExtension/web/library/layoutSpecs.json

```diff
@@ -248,7 +248,7 @@
               "elementType": "group",
               "include": [
                 {
-                  "path": "Core.List.List.Create"
+                  "path": "Core.List.List Create"
```

Comment:
@QilongTang do we need to make these same fixes in revit layout spec now?

Reply:
@mjkkirschner I believe these nodes are Dynamo specific, the Revit layoutspec only contains Revit nodes and will be appended to the Dynamo main one, let me double check

Reply:
@mjkkirschner Confirmed. No need